repository/src/main/resources/db/changelog/db.changelog-0.0.13.xml

```diff
@@ -3,8 +3,9 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <changeSet id="13" author="ccd@hmcts.net">
+        <validCheckSum>8:cecbe4dfadf0450889899e86fde0f849</validCheckSum>
         <createTable tableName="case_type_user_role">
-            <column name="id" type="serial">
+            <column autoIncrement="true" name="id" type="serial">
```

Comment:
for my knowledge, how come in this PR we are adding directly `autoIncrement="true"` rather than using the approach we used in other PRs, i.e.:
`<addAutoIncrement>`? are the 2 approaches equivalent?